ext/standard/basic_functions.stub.php

```diff
@@ -66,3 +66,7 @@ function base64_encode(string $str): string {}
 
 /** @return string|false */
 function base64_decode(string $str, bool $strict = false) {}
+
+/* crypt.c */
+
+function crypt(string $str, string $salt = null): string {}
```

Comment:
```suggestion
function crypt(string $str, string $salt = UNKNOWN): string {}
```

`null` isn't an allowed value here, and the function treats this input specially (it generates a notice if not passed).

Reply:
thanks for pointing that out, I saw the notice but didn't really known what to use here as replacement